MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -1681,7 +1681,7 @@ private void stgopt()
 
 #if DEBUG
                 _logger.Info(
```

Comment:
We can probably make this `Debug` -- I don't think they'd be needed by an ordinary use at `Info` level.

Reply:
And all the `opt` ordinals for that matter.